docs/configuration/entrypoints.md

```diff
@@ -1,4 +1,4 @@
-# Entry Points Definition
+﻿# Entry Points Definition
```

Comment:
I don't see any change on this line, could you restore the previous state of this line.

Reply:
Couldnt see where this was injected in dev tools, have tweaked via github to remove